launch/launch/frontend/expose.py

```diff
@@ -98,7 +98,15 @@ def expose_impl_decorator(exposed):
                     name
                 )
             )
-        parse_methods_map[name] = found_parse_method
+        if exposed_type == 'action':
+            # For actions, validate that the user didn't provide unkown attributes or children
```

Comment:
```suggestion
            # For actions, validate that the user didn't provide unknown attributes or children
```

Reply:
Fixed in https://github.com/ros2/launch/pull/468/commits/09871dd4925a29a27ac8f6b67a7add84f120d7c8.